option java_package = "edu.calpoly.capstone.sensor";
option java_outer_classname = "Tcas";
message TcasReport {
    required sfixed32 id = 1; // ID of target
    required sfixed32 range = 2; // relative distance
    required sfixed32 altitude = 3; // relative altitude
    required sfixed32 bearing = 4; // relative heading
}